### PR TITLE
Update Requirments

### LINKED ISSUE #237: Update requirements
we are using old versions of requirements also some other repos it would be nice to see how much we can upgrade and upgrade to the latest.

### PR DESCRIPTION
**Pull request by nemaniarjun**
Fixes #237

### CODE REVIEW

**Comment by abhsag24** [comment]
Hi thank you for your Patch make sure you have the right commit structure that is 
```
tag: Shortlog

body
Fixes <issue number/url>
```

File: requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-bs4~=0.0.1
-requests~=2.11.1
+beautifulsoup4~=4.5.1
```
Comment (abhsag24) [inline requirements.txt]: doesnt this need a changing of imports?

Comment (nemaniarjun) [inline requirements.txt]: I don't think so. Everything works fine and bs4 just downloads Beautiful Soup 4.. Also pytest passes. So...

Comment (abhsag24) [inline requirements.txt]: point noted.

**Pull request merged**